doc/contributing/maintaining-openssl.md

```diff
@@ -9,7 +9,8 @@ currently need to generate four PRs as follows:
   below for OpenSSL 3.x.x.
 * a PR for 16.x following the instructions in the v16.x-staging version
   of this guide.
-* a PR for 14.x following the instructions in the v14.x-staging version
+* a PR for 14.x following the instructions in the \[v14.x-staging version]
+  (<https://github.com/nodejs/node/blob/v14.x-staging/doc/guides/maintaining-openssl.md>)
```

Comment:
This is being rendered as 
![image](https://user-images.githubusercontent.com/5445507/196695152-35746b51-8921-4ebb-a703-9328c9b9dc67.png)
due to the whitespace between `[...]` and `(...)`.